src/App.js

```diff
@@ -5,13 +5,16 @@ import Navbar from "./components/Navbar/Navbar";
 import ProductCard from "./components/ProductCard/ProductCard";
 import { useEffect, useState } from "react";
 import axios from "axios";
-import CategoryCard from "./components/CategoryCard/CategoryCard";
-import HeroImg from "./components/HeroImg/HeroImg";
-import CartCard from "./components/CartCard/CartCard";
+import Home from "./pages/Home/Home";
+import Product from "./pages/Product/Product";
 import Footer from "./components/Footer/Footer";
+import Wishlist from "./pages/Wishlist/Wishlist";
+import Cart from "./pages/Cart/Cart";
+import Signup from "./pages/Auth/Signup";
+import Login from "./pages/Auth/Login";
```

Comment:
Make an index.js file in your components folder and export all these from that file